courses/management/commands/gen_fake_data.py

```diff
@@ -53,6 +62,9 @@ def handle(self, *args, **options):
                 course_runs_range = range(randint(1, int(options['course_runs'])))
                 for _ in course_runs_range:
                     record_count += 1
-                    CourseRunFactory.create(course=course)
+                    run = CourseRunFactory.create(course=course)
+                    if options['course_prices']:
+                        CoursePriceFactory.create(course_run=run)
```

Comment:
We should also set `is_valid=True` here. Also, all course runs need to have the same price